manubot/cite/tests/test_doi.py

```diff
@@ -23,9 +25,13 @@ def test_expand_short_doi_not_short():
         expand_short_doi("10.1016/S0933-3657(96)00367-3")
 
 
-def test_get_doi_csl_item_crosscite():
+@pytest.mark.parametrize(
+    "get_doi_csl_item_negotiation",
```

Comment:
Could it be confusing to have the argument name be the same as the manubot.cite.doi function name?

Reply:
Perhaps, but get_doi_csl_item_datacite and get_doi_csl_item_crosscite are really just versions of get_doi_csl_item_negotiation with different default argument values.